empirical-org/Empirical-Core
# Build and deploy docker image to quill-lms-sprint-docker heroku app

## WHAT
Dockerfile and build and deploy scripts for quill-lms-sprint-docker

Steps for testing:
1) Download and install Docker Desktop for Mac (requires creating an account): https://hub.docker.com/editions/community/docker-ce-desktop-mac

2) Run the following:
```
cd services/QuillLMS
time ./build_and_deploy_heroku_docker_image.sh sprint
```
3) Verify that the deploy succeeded by visiting: https://quill-lms-sprint-docker.herokuapp.com/

4) Try running the command again to verify that it runs faster on subsequent runs:
```
time ./build_and_deploy_heroku_docker_image.sh sprint
```

## WHY
We want to speed up our LMS deploy

## HOW
Building a docker image and deploying it to heroku

## Have you added and/or updated tests?
(The answer should mostly be 'YES'. If you answer 'NO', please justify.)
NO
